.gitignore

```diff
@@ -20,3 +20,4 @@ Build
 *.swp
 *.orig
 .arcconfig
+
```

Comment:
This file doesn't need to be changed, and the addition of a newline doesn't add any value. Can you revert the file to what it was beforehand?